dts/client.py

```diff
@@ -11,13 +11,14 @@
 from urllib.error import (
     HTTPError,
 )
+from typing import Any
 
 logger = logging.getLogger('dts')
 api_version = 1
 
 class KBaseAuth(AuthBase):
     """Attaches a KBase-sensible Authorization header to the given Request object."""
-    def __init__(self, api_key):
+    def __init__(self, api_key: str):
```

Comment:
`self` can be given the type `"KBaseAuth"` and the output of the function is `None`, i.e.

```py
def __init__(self: "KBaseAuth", api_key: str) -> None:
```

Reply:
(self can be assigned the type `"KBaseAuth"` throughout the class)

Reply:
cheesy, but good to know!